dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NotificationIT.java

```diff
@@ -45,6 +50,15 @@ private Notification anotherTestNotification() {
         return notification;
     }
 
+    private Notification longNotification() throws IOException {
+        Notification notification = new Notification();
+        String message = FileUtils.readFileToString(new File(ResourceHelpers.resourceFilePath("longNotification.txt")), StandardCharsets.UTF_8);
```

Comment:
You can use Java 11's `Files.readString(Paths.get(ResourceHelpers.resourceFilePath("longNotification")));` instead of Apache commons

Reply:
I changed it to the suggestion, what are the advantages of this approach as opposed to the Apache commons?

Reply:
Don't need to use a 3rd party library when it can be done natively. One day we might be able to remove the 3rd party library entirely